projectroles/tests/test_ui.py

```diff
@@ -1688,6 +1693,37 @@ def test_settings_label_icon(self):
         ).find_element(By.TAG_NAME, 'svg')
         self.assertTrue(logo.is_displayed())
 
+    def test_submit_button(self):
+        """Test rendering of submit button"""
+        url = reverse(
+            'projectroles:create', kwargs={'project': self.category.sodar_uuid}
+        )
+        self.login_and_redirect(
+            self.superuser, url, wait_elem=None, wait_loc='ID'
+        )
+        element = self.selenium.find_element(
+            By.CLASS_NAME, 'sodar-pr-btn-submit-once'
+        )
+        self.assertEqual(element.text, 'Create')
+
+        self.assertTrue(element.is_enabled())
+
+        # Define timeout and retry interval
+        timeout = 10  # Maximum time to wait in seconds
+        retry_interval = 0.5  # Time between retries in seconds
+
+        start_time = time.time()
+        element.click()
+        while True:
```

Comment:
This is looking good, I have some ideas for further improvement:

- Instead of calculating time, we can just set a number of retries, iterate with `range()` and `break` once condition is met. This would be more compact while doing the same thing.
- 0.5 seconds might be a bit too much to wait. I feel this operation might execute faster than that in some cases.. Maybe 0.2s would be better?
- As final cleanup, make sure to remove unneeded blank lines :)